src/main/java/codeu/controller/ActivityServlet.java

```diff
@@ -61,14 +68,21 @@ void setUserStore(UserStore userStore) {
   void setConversationStore(ConversationStore conversationStore) {
     this.conversationStore = conversationStore;
   }
+  
+  void setMentionStore(MentionStore mentionStore) {
+	this.mentionStore = mentionStore;
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
 
     List<Conversation> conversations =
         new ArrayList<Conversation>(conversationStore.getAllConversations());
+    List<Mention> mentions = 
+    	new ArrayList<Mention>(mentionStore.getAllMentions());
     Collections.sort(conversations, (a, b) -> b.getCreationTime().compareTo(a.getCreationTime()));
+    Collections.sort(mentions, (a, b) -> b.getCreationTime().compareTo(a.getCreationTime()));
     request.setAttribute("conversations", conversations);
```

Comment:
Don't forget you'll need to add a request.setAttribute("mentions", mentions); in here as well.